### PR TITLE
Improve UI for PoiInfoFragment

### PR DESCRIPTION
**Pull request by Nico281102**
# Description

Enhance 'PoiInfoFragment' UI for improved user experience.
# How Has This Been Tested?

- [x] Manually
- [ ] Unit tests
- [ ] Integration tests

# Screenshots / Videos

Screenshot_20231215_140808

### CODE REVIEW

**Comment by NeverMendel** [comment]
Any update?

File: app/src/main/res/layout/fragment_poi_info.xml [hunk 1]
```diff
@@ -1,39 +1,122 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="?android:attr/windowBackground"
```
[the remark this answers is not in the document]
Comment (Nico281102) [reply app/src/main/res/layout/fragment_poi_info.xml:8]: You don't use  android:background in poi_list_item.xml, are you saying that i don't have to use that attribute?

**Comment by NeverMendel** [comment]
@Nico281102 Could you format the time and distance properly? Max 1 significant digit and correct unit (ex. km or m depending if distance is less than 1000m)

Also can you add the two feedback buttons? I think some they might look nice with `#1DA1F2` color or something similar

**Comment by Nico281102** [comment]
> @Nico281102 Could you format the time and distance properly? Max 1 significant digit and correct unit (ex. km or m depending if distance is less than 1000m)
> 
> Also can you add the two feedback buttons? I think some they might look nice with `#1DA1F2` color or something similar

First of all, as you can see, i've just fixed the time and distance formats. Talking about the buttons:
Do you wanna use the same color for two different buttons? What am I going to write inside the two buttons? could it be better if we use stars to send feedback instead?

**Comment by NeverMendel** [comment]
> Do you wanna use the same color for two different buttons? What am I going to write inside the two buttons? could it be better if we use stars to send feedback instead?

Yeah, same color should be good. Up to you what to write, come up with a suggestion ;)
I think two buttons would work better. Have a look at the docs or let's chat offline on that if you'd like.

Also, put a thumbs up and down icon in the button (other than the text ofc)

File: app/src/main/java/com/appocalypse/naturenav/poiinfo/PoiInfoFragment.java [hunk 2]
```diff
@@ -24,22 +32,56 @@ public class PoiInfoFragment extends Fragment {
...
+        binding.poiAddressTextView.setText(getString(R.string.address) + " " + poi.address);
+
+        updateDistanceTextView(binding.poiAirDistanceTextView, poi.airDistanceMeters, R.string.air_distance_placeholder);
+        updateDistanceTextView(binding.poiRoadDistanceTextView, poi.roadDistanceMeters, R.string.road_distance_placeholder);
+
+        // convert seconds to minutes
+        binding.poiDurationTextView.setText(getString(R.string.duration_placeholder, formatOneSignificantDigit(poi.roadDistanceSeconds / 60.0)));
+    }
+
+    private void updateDistanceTextView(TextView textView, double distanceMeters, @StringRes int stringResourceId) {
```
Comment (NeverMendel) [inline app/src/main/java/com/appocalypse/naturenav/poiinfo/PoiInfoFragment.java:71]: IMO it's better to mantain this logic separated from the TextView. We can just format the last part of the text and append it later on

**Comment by NeverMendel** [comment]
@Nico281102 I made some improvement. Please have a look.

Still missing the two buttons, but will do in a separate PR

**Review by NeverMendel — APPROVED** [review]

**Pull request merged**